### PR TITLE
FEAT: Allow Script execution in PHP runtime

### PR DESCRIPTION
**Pull request by JustinBack**
This PR removes the --no-script parameter so custom extensions can be installed through the composer.json

e.g.

```sh
    "scripts": {
        "pre-package-install": "apk update && docker-php-ext-install pdo"
    }
```

Important in an environment where no custom dockerfiles can be used.

### CODE REVIEW

**Review by Meldiron — APPROVED** [review]
Can't see a reason to not allow it. Ill wait for 1 more review before merging, as it is security related.

**Comment by JustinBack** [comment]
Âwesome, security wise there isnt any difference as to nodejs runtimes as scripts are executed there as well ;)

**Comment by Meldiron** [comment]
Thanks to Open Runtimes v3, build and start commands are now fully customizable ✨

More info: https://github.com/open-runtimes/open-runtimes/pull/177

Closing this PR as it has been addressed by v3.

**Pull request closed without merging**
